src/components/Header.js

```diff
@@ -0,0 +1,27 @@
+import Create from './MakeElement';
+
+const Header = () => {
+  const header = Create.element('HEADER');
+  const ul = Create.element('ul');
+  const h2 = Create.element('H2');
+  const h2Text = Create.text('Burger Stop');
+  const nav = Create.element('NAV');
+
+  header.appendChild(h2);
+  header.appendChild(nav);
```

Comment:
You could also use [`append`](https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/append) instead of `appendChild`:

```js
element.append(child1, child1, child3, ...)
```

Reply:
Thanks for the feedback, I've refactored the code